drivers/vdpa/virtio/virtio_vdpa.c

```diff
@@ -474,6 +474,25 @@ virtio_vdpa_dev_notifier_work(void *arg)
 	return ret;
 }
 
+static void virtio_vdpa_doorbell_relay_disable(struct virtio_vdpa_priv *priv)
+{
+
+	uint16_t nr_virtqs = priv->hw_nr_virtqs;
+	int vq_idx, ret;
+
+	for(vq_idx = 0; vq_idx < nr_virtqs; vq_idx++) {
+		if (priv->vrings[vq_idx]->notifier_state == VIRTIO_VDPA_NOTIFIER_RELAY_ENABLED) {
+			ret = virtio_vdpa_virtq_doorbell_relay_disable(priv, vq_idx);
+			if (ret) {
+				DRV_LOG(ERR, "%s doorbell relay disable failed ret:%d",
+								priv->vdev->device->name, ret);
+			}
+			priv->vrings[vq_idx]->notifier_state = VIRTIO_VDPA_NOTIFIER_RELAY_DISABLED;
```

Comment:
the state seems to be set in virtio_vdpa_virtq_doorbell_relay_disable already. so this is not needed?

Reply:
done